nixos/modules/services/misc/octoprint.nix

```diff
@@ -106,6 +106,8 @@ in
 
     systemd.tmpfiles.rules = [
       "d '${cfg.stateDir}' - ${cfg.user} ${cfg.group} - -"
+      # this will allow octoprint read access to raspberry specific hardware to check for throttling
+      "a /dev/vchiq - - - - u:octoprint:r"
```

Comment:
Please test if it works on a rpi and does not cause failures on what's not a rpi (where there is no /dev/vchiq), especially the status of `systemd-tmpfiles-setup.service`.

Reply:
testing right now

Reply:
My testing showed:

- No errors on x86-linux systems (tested in a VM) where no `dev/vchiq` exists. `systemctl status systemd-tmpfiles-setup.service` does not show any errors, just like `journalctl -xeu systemd-tmpfiles-setup.service`
- No errors show on raspberry pi 4 (just like above)
- correct permissions are added as confirmed by `getfacl /dev/vchiq` 
- `octoprint` can use `vcgencmd` and confirmed per CLI with `sudo -u octoprint vcgencmd get_throttled`
- `octoprint` WebUI doesn't show `vcgencmd` error anymore and seems to detect correct sate (as seen in startup logs)